src/bundle.js

```diff
@@ -0,0 +1,46 @@
+import Mfs from 'memory-fs'
+import webpack from 'webpack'
```

Comment:
Given the trouble that we've had with webpack, we may want to evaluate rollup as well.   Or make this pluggable.